app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -64,7 +66,8 @@ export default function ResourceAuthoringPage() {
       (identifierValue.trim() !== '' && identifierSystem !== savedIdentifierSystem) ||
       name !== (resource?.name ?? '') ||
       title !== (resource?.title ?? '') ||
-      description !== (resource?.description ?? '')
+      description !== (resource?.description ?? '') ||
+      (resource?.resourceType === 'Measure' && (library ? library !== resource?.library?.[0] : resource?.library?.[0]))
```

Comment:
```suggestion
      (resource?.resourceType === 'Measure' && (library ? library !== resource.library?.[0] : resource.library?.[0]))
```
I think we know `resource` exists because of the first part of this line